1

Daria2608 opened the pull request:
DailyInfo секція, виконано:
 Зміна тексту в компоненті <ChooseData>,  в залежності від того, що обере користувач в календарі.
2. стилізація ChooseData
3. AddButton, стилізація

Daria2608 committed:
1

DailyInfo секція, виконано:
 Зміна тексту в компоненті <ChooseData>,  в залежності від того, що обере користувач в календарі.
2. стилізація ChooseData
3. AddButton, стилізація

melnykartem3 reviewed (dismissed):
на десктопі не видно бар, треба додати стилізацію до всіх пристроїв,але логіка правильна

Daria2608 committed:
2

1. стилізація ChooseData + AddBtn для всіх екранів
2. Приховування компонента ChooseData/AddBtn в залежності від розміру екрану при взаємодії користувача з UserBar

Andriana-Petrenko committed:
Merge branch 'main' of https://github.com/melnykartem3/aquatrack-project-frontend into DailyInfo
